fbw-a32nx/src/systems/instruments/src/EFB/Store/features/globalSettings.ts

```diff
@@ -0,0 +1,34 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+//Store for alle settings which should be application global for the EFB application but not neccesary for the simvars
```

Comment:
```suggestion
// Store for all settings which should be global for the EFB but not neccesary for the simvars.
```

Reply:
I already thought that, but the system needs a store where it can save the setting if the user want cabin sounds at all. 
Currently it is using the simvar which disables the sound as storing variable. When the system enables the sound because there are more than 0 passengers on board the simvar setting will be overwritten and the user who dont want any cabin sounds gets cabin sounds.  In the globalSettings i store the information if the user want cabin sounds at all.  Sorry for my bad spelling in the inline comments.

Reply:
There is no `SimVar` available to store it ?

Like `EFB_AMBIANCE_SOUND_ENABLED` ?

(Don't worry for the the spelling, we all make mistakes)